src/Core/src/Platform/iOS/KeyboardAutoManagerScroll.cs

```diff
@@ -608,6 +694,35 @@ internal static nfloat FindKeyboardHeight()
 		return window.Frame.Height - kbSize.Height;
 	}
 
+	// In the case we have PrefersLargeTitles set to true, the UINavigationBar
+	// has additional height that collapses when scrolling. Try to remove
+	// this collapsable height difference from the calculated move distance.
+	static nfloat AdjustForLargeTitles(nfloat move, UIScrollView superScrollView, UINavigationController navController)
+	{
+		var navBarCollapsedHeight = 44;
+		var minPageMoveToCollapseNavBar = 52;
```

Comment:
I wonder if these values can change with the device - probably better to see if I can get values instead of hard coding